### PR TITLE
[W6.1h][W11-B1] Li JiaYao

### PR DESCRIPTION
**Pull request by JYL123**
Add a Printable interface so that:

1. Override the getPrintableString in classes Name, Phone, Email, and Address so that each produces a printable string representation of the object.

2. Add public String getPrintableString(Printable... printables) in Person

### CODE REVIEW

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address implements Printable{
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Address.java]: need space between `Printable {`

File: src/seedu/addressbook/data/person/Printable.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package seedu.addressbook.data.person;
+
+/**
+ * A Printable interface for a Person's particular (e.g. email, phone, address, name) in the addressbook.
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Printable.java:4]: :+1:

**Comment by okkhoy** [comment]
@JYL123 
Some comments added. Please ack & close the PR after reading comments.

**Comment by JYL123** [comment]
Changes are made. SourceTree cannot detect a space change so no push here. Thanks for reviewing. PR is closed now.

**Comment by JYL123** [comment]
The SourceTree is too slow in detecting sometimes.

**Pull request closed without merging**